spyderlib/widgets/sourcecode/codeeditor.py

```diff
@@ -59,8 +59,12 @@
 from spyderlib.py3compat import to_text_string
 
 try:
-    import IPython.nbformat as nbformat
-    from IPython.nbconvert import PythonExporter as nbexporter
+    try:   # Ipython 4
+        import nbformat as nbformat
+        from nbconvert import PythonExporter as nbexporter  
+    except:  # Ipython 3

```

Comment:
Thanks for this @stonebig 

But we should catch the actual exception, like `except ImportError?`

Also the comments